claimManagement/src/main/java/org/openimis/imisclaims/ImisActivity.java

```diff
@@ -233,6 +235,8 @@ private AlertDialog showLoginDialogBox(Runnable onLoggedIn, Runnable onCancel) {
         return alertDialogBuilder.show();
     }
 
+    protected void onUserLoggedIn(){}
```

Comment:
It's a way for me to have a method called when the user is logged in, which the sub-classes can override to take action. Since the base class doesn't have to do anything on login, the method here is empty.